test/test_foreach.py

```diff
@@ -1054,25 +1054,32 @@ def test_addcdiv(self, device, dtype):
                 return
         self._test_pointwise_op(device, dtype, torch._foreach_addcdiv, torch._foreach_addcdiv_, torch.addcdiv)
 
-    @dtypes(*torch.testing.get_all_dtypes(include_bfloat16=False, include_bool=False, include_complex=False))
+    @dtypes(*torch.testing.get_all_dtypes())
```

Comment:
What added bfloat16 and bool support? Were those supported all along?

Reply:
I've added support for torch.bool and torch.bfloat16 in this PR